public/locales/cze/user-profile.json

```diff
@@ -1,5 +1,16 @@
 {
   "Member since {{date, MMM Do, YYYY}}": "Členem od {{date, Do MMM YYYY}}",
   "Online {{date, fromNow}}": "Online {{date, fromNow}}",
-  "Online long ago": "Online dávno"
+  "Online long ago": "Online dávno",
```

Comment:
It's nice to have these translations which we can use for testing.

Anyways this is something that we don't have to do. We can leave it for people, who will be translating (when we figure out nice tools for that).